spec/system/sign_out_spec.rb

```diff
@@ -10,7 +10,7 @@
       sign_in @user
       visit ideas_path
       click_link 'Sign out'
-      expect(page).to have_content 'Log in'
+      expect(page).to have_content 'Sign in to Great Ideas'
```

Comment:
@Pareis2 issue was here, you updated the title of the log in page, but the test was still looking for the old title :)